### PR TITLE
Update for 1.0.1 dotnet core collection.

### PR DESCRIPTION
**Pull request by jerboaa**
Includes the work-around for https://github.com/dotnet/corefx/issues/12758

### CODE REVIEW

**Comment by jerboaa** [comment]
@omajid @dbhole: Could you have a look, please? Thanks!

**Comment by omajid** [comment]
Looks good to me.

File: 1.0/Dockerfile.rhel7 [hunk 1]
```diff
@@ -52,14 +52,20 @@ USER 1001
...
+#   in nuget packages. See:
+#   https://github.com/dotnet/corefx/issues/12758
 RUN cd /opt/app-root/src && mkdir cache-warmup && \
     cd cache-warmup && \
     scl enable rh-dotnetcore10 -- dotnet new && \
     cd .. && \
     rm -rf cache-warmup && \
-    rm -rf /tmp/NuGetScratch
+    rm -rf /tmp/NuGetScratch && \
+    find .nuget -name '*.cs' -exec rm '{}' \;
```
Comment (omajid) [inline 1.0/Dockerfile.rhel7]: No need. It's not worth a rebuild.

**Comment by jerboaa** [comment]
Thanks for the review!

**Comment by omajid** [comment]
Anytime!

**Comment by jerboaa** [comment]
[skipci]

**Pull request merged**